docs/modules/ROOT/pages/platforms/grafana.adoc

```diff
@@ -0,0 +1,409 @@
+  -d @redis-enterprise-cluster-overview.json
+----
+
+==== Provisioning Dashboards
+
+Create `/etc/grafana/provisioning/dashboards/redis-enterprise.yml`:
+
+[source,yaml]
+----
+apiVersion: 1
+
+providers:
+  - name: 'Redis Enterprise'
+    orgId: 1
+    folder: 'Redis Enterprise'
+    type: file
+    disableDeletion: false
+    updateIntervalSeconds: 30
+    allowUiUpdates: true
+    options:
+      path: /var/lib/grafana/dashboards/redis-enterprise
+----
+
+Place dashboard JSON files in `/var/lib/grafana/dashboards/redis-enterprise/`.
+
+== Example Configurations
+
+=== Production-Ready Templates
+
+Complete dashboard and configuration files are available in the GitHub repository:
+
+* https://github.com/redis-field-engineering/redis-enterprise-observability/tree/main/grafana_v2[Grafana Dashboards^] - Full JSON dashboard definitions
+** Cluster overview dashboards
+** Database performance dashboards
+** Node monitoring dashboards
+** Active-Active replication dashboards
+
+* https://github.com/redis-field-engineering/redis-enterprise-observability/tree/main/grafana_v2/demo_v2[Complete Demo Environment^] - Working docker-compose setup
+** https://github.com/redis-field-engineering/redis-enterprise-observability/blob/main/grafana_v2/demo_v2/docker-compose.yml[docker-compose.yml^] - Full stack (Prometheus + Grafana + Redis)
+** https://github.com/redis-field-engineering/redis-enterprise-observability/blob/main/grafana_v2/demo_v2/prometheus.yml[prometheus.yml^] - Pre-configured scrape settings
+
+* https://github.com/redis-field-engineering/redis-enterprise-observability/tree/main/grafana_v2/kickstart_v2[Kickstart Environment^] - Quick production setup
+
+TIP: Import the complete dashboard JSON files directly into Grafana rather than recreating panels manually. All dashboards include pre-configured variables and alert templates.
+
+== Available Dashboards
+
+See the xref:dashboards/catalog.adoc[Dashboard Catalog] for the complete list of Grafana dashboards.
+
+== Configuration
+
+=== Dashboard Variables
+
+Most dashboards include template variables for dynamic filtering.
+
+==== Creating Variables
+
+Example variable configuration for cluster selection:
+
+[source,text]
+----
+Name: cluster
+Label: Cluster
+Type: Query
+Data source: Redis-Enterprise-Prometheus
+Query: label_values(redis_enterprise_cluster_health, cluster)
+Regex: .*
+Multi-value: true
+Include All option: true
+----
+
+Example variable for database selection:
+
+[source,text]
+----
+Name: database
+Label: Database
+Type: Query
+Data source: Redis-Enterprise-Prometheus
+Query: label_values(redis_enterprise_database_up{cluster="$cluster"}, database)
+Refresh: On Dashboard Load
+Multi-value: true
+Include All option: true
+----
+
+==== Using Variables in Queries
+
+Use variables in your panel queries:
+
+[source,promql]
+----
+# Filter by selected cluster and database
+redis_enterprise_database_ops_per_sec{cluster="$cluster", database="$database"}
```

Comment:
`redis_etnerprise_databse_ops_per_sec` is not a metric - something useful to filter on might be something like `sum by(cluster,db) irate(endpoint_read_reqeusts{cluster="$cluster", db="$db"}[1m])` - need to double check the promql syntax there.